src/UserSpaceInstrumentation/Trampoline.cpp

```diff
@@ -85,23 +85,29 @@ ErrorMessageOr<std::vector<AddressRange>> GetTakenAddressRanges(pid_t pid) {
 }
 
 ErrorMessageOr<AddressRange> FindAddressRangeForTrampoline(
-    const std::vector<AddressRange>& taken_ranges, const AddressRange& code_range, uint64_t size) {
+    const std::vector<AddressRange>& unavailable_ranges, const AddressRange& code_range,
+    uint64_t size) {
   constexpr uint64_t kMax32BitOffset = 0x7fffffff;
+  constexpr uint64_t kMax64BitAdress = 0xffffffffffffffff;
```

Comment:
is it UINT64_MAX? :) It is very hard to count number of bytes this way, maybe separate every 4 bytes/digits with `'`?

Reply:
done